### ISSUE TITLE
Error in color extractor modifying color target

### ISSUE THREAD

**Issue by czfadmin** [microsoft/vscode#115512]
Issue Type: <b>Bug</b>

When I want to change the color of "tab. ActiveBorderTop", the color extractor also displays the message about "tab. ActiveBorderTop", but it actually changes one of the target colors in the image below. But It works again when I reload VSCode.
![image](https://user-images.githubusercontent.com/13393959/106413744-d9a79a80-6485-11eb-994b-8f1c51913e24.png)

VS Code version: Code - Insiders 1.53.0-insider (7b0cfdd04ad530a9c8d782c618afd79290be3d64, 2021-01-27T08:59:32.031Z)
OS version: Windows_NT x64 10.0.19042
Remote OS version: Linux x64 5.9.14-arch1-1

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-9400H CPU @ 2.50GHz (8 x 2496)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.79GB (0.96GB free)|
|Process Argv|--crash-reporter-id 8f715c61-2e61-4b8e-a557-7077b69c8983|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: archlinux|
|OS|Linux x64 5.9.14-arch1-1|
|CPUs|Intel(R) Core(TM) i5-9400H CPU @ 2.50GHz (2 x 2496)|
|Memory (System)|1.94GB (0.28GB free)|
|VM|100%|
</details><details><summary>Extensions (49)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|2.1.0
project-manager|ale|12.0.1
nestjs|ash|1.4.2
bracket-pair-colorizer-2|Coe|0.2.0
es7-react-js-snippets|dsz|3.1.0
vscode-ionic-snippets|fiv|2.2.2
auto-close-tag|for|0.5.10
auto-rename-tag|for|0.1.6
vscode-pull-request-github|Git|0.22.0
vscode-peacock|joh|3.9.1
Lisp|mat|0.1.11
remote-containers|ms-|0.157.0
remote-ssh|ms-|0.63.0
remote-ssh-edit|ms-|0.63.0
remote-wsl|ms-|0.53.2
vscode-remote-extensionpack|ms-|0.20.0
github-browser|ms-|0.0.14
vscode-github-issue-notebooks|ms-|0.0.60
debugger-for-chrome|msj|4.12.12
bootstrap4-vscode|the|6.1.0
vscode-icons|vsc|11.1.0
material-theme|zhu|3.9.13
rainbow-brackets|2gu|0.0.6
Bookmarks|ale|12.1.4
vscode-sqlite|ale|0.11.1
vscode-django|bat|1.3.0
python-preview|don|0.0.4
prettier-vscode|esb|5.8.0
vscode-pull-request-github|Git|0.22.0
vscode-test-explorer|hbe|2.19.5
rest-client|hum|0.24.4
vscode-python-test-adapter|lit|0.6.6
MagicPython|mag|1.1.0
git-graph|mhu|1.28.0
python|ms-|2021.1.502429796
vscode-pylance|ms-|2021.1.3
jupyter|ms-|2020.12.414227025
vscode-typescript-next|ms-|4.2.20210131
sqltools|mtx|0.23.0
sqltools-driver-mysql|mtx|0.2.0
sqltools-driver-pg|mtx|0.2.0
sqltools-driver-sqlite|mtx|0.2.0
live-sass|rit|3.0.0
LiveServer|rit|5.6.1
sass-indented|syl|1.8.15
pdf|tom|1.1.0
vscodeintellicode|Vis|1.2.11
eno|Wsc|2.3.53
markdown-pdf|yza|1.4.4

(2 theme extensions excluded)

</details>

**Comment by gjsjohnmurray** [comment]
Are you able to reproduce the issue after reloading with all extensions disabled? If not, try extension bisect command to identify the culprit.

**Comment by aeschli** [comment]
Did you see this more than once? Does it just start happening after editing?
If you have reproducible steps, it would be great if you can provide them along with the settings file content

**Comment by aeschli** [comment]
Duplicate of #115415

**Comment by czfadmin** [comment]
@gjsjohnmurray 
I repeat the situation, in the theme of the work interval, if there is a comment in the list when you customize the color, after saving, this phenomenon will occur. After the comment is removed, the phenomenon still exists. This can only be eliminated by overloading VSCode.

![image](https://user-images.githubusercontent.com/13393959/106537966-5a25d400-6536-11eb-93ff-b1f259880125.png)
![image](https://user-images.githubusercontent.com/13393959/106538195-db7d6680-6536-11eb-97a4-7725d51a364c.png)

I also repeated the problem with all plug-ins disabled at all
![image](https://user-images.githubusercontent.com/13393959/106538535-58a8db80-6537-11eb-9b6e-5eb946e42e24.png)

**Comment by aeschli** [comment]
@czfadmin Is the message wrong or is the color picker icker? Where is the mouse pointer when you see it?
Can you make a screen capture?

**Comment by czfadmin** [comment]
@aeschli 
 Sorry. I described a bit of a problem above, and now I find that it is not necessary to have a comment in the list to cause this phenomenon to occur, and this phenomenon will also occur when there is no comment. Below is the GIF I operated with all extensions disabled.

![error](https://user-images.githubusercontent.com/13393959/106606462-20d47f00-659d-11eb-8dce-0962610b5319.gif)
But I noticed this phenomenon.
![image](https://user-images.githubusercontent.com/13393959/106606542-38136c80-659d-11eb-9e88-cf1511ba9c7b.png)
